librz/core/cmd_windows_heap.c

```diff
@@ -0,0 +1,35 @@
+#include <rz_core.h>
+
+RZ_IPI RzCmdStatus rz_cmd_debug_process_heaps_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
```

Comment:
i'm tempted to recommend  having two parts in this file, one for windows and one for everything else.

```C
#if __WINDOWS__
RZ_IPI RzCmdStatus rz_cmd_debug_process_heaps_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
        ...
}
...
#else
RZ_IPI RzCmdStatus rz_cmd_debug_process_heaps_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
        eprintf("Windows heap parsing is not supported on this platform\n");
	return RZ_CMD_STATUS_ERROR;
}
...
#endif
```

Reply:
it actually makes sense.